apis/apps/v1alpha1/containerrecreaterequest_types.go

```diff
@@ -160,6 +160,8 @@ type ContainerRecreateRequestContainerRecreateState struct {
 	Phase ContainerRecreateRequestPhase `json:"phase"`
 	// A human readable message indicating details about this state.
 	Message string `json:"message,omitempty"`
+	// The number of times the container has been restarted by krusie-daemon
```

Comment:
How about 
`// The number of times the container has been restarted by krusie-daemon` =>  `// The restart counts of the container has been restarted by krusie-daemon`